packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -53,17 +53,25 @@ The widget upgrade for a redirect sign-in flow depends on whether you configured
 
 To update the widget:
 
-1. In the Admin Console, go to **Customizations** > **Sign-in page code editor**.
+> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
 
-   In the **Okta Sign-In Widget Version** section, check that the **Major Version** is the highest version available and **Minor Version** is **latest**. The widget is always the latest version if you're not using a custom URL domain.
+1. In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the **Sign-in page** section. (EA users: go to **Customizations** > **Brands**, and then select the brand you want.)
 
-2. If you're using the [custom domain feature](/docs/guides/custom-url-domain/) and the version isn't correct, you can change the widget's version. Click **Edit** in the **Okta Sign-In Widget Version** section and then select the **Major Version** and **Minor Version** fields.
+1. Go to the **Settings** tab. (EA users: in the **Pages** tab, click **Configure** for the page that has the version you want to upgrade.)
 
-3. Click **Save** at the bottom of the page.
+  - In the **Sign-In Widget Version** section, check that the **Version** is the highest version available. The widget is always the latest version if you're not using a custom URL domain.
+  - If you're using the [custom domain feature](/docs/guides/custom-url-domain/) and the version isn't correct, you can pin the widget's version. Click **Edit** in the **Sign-In Widget Version** section and then select the **Version** field.
+  - (EA users: You can't customize the Sign-In Widget for the Okta default brand.)
 
-4. Test your app's supported authentication sign-up and recovery flows to ensure that they still work.
+  > **Note:** When the third generation of the Sign-In Widget is enabled, the pinned version is ignored.
 
-5. Verify the CSS and localization changes that you made are reflected in the new version.
+1. Click **Save to draft**.
+
+1. Test your app's supported authentication sign-up and recovery flows to ensure that they still work.
```

Comment:
How is this done? As far as I know, we only support previewing the draft sign-in page but I don't think you can actually go through the flows. Is it a feature of the new SIW?

Reply:
Good point. I'll work with you to update this step.

Reply:
Updated.